subprojects/plugins/src/main/groovy/org/gradle/api/internal/tasks/testing/filter/DefaultTestFilter.java

```diff
@@ -25,7 +25,8 @@
 
 public class DefaultTestFilter implements TestFilter {
 
-    private Set<String> testNames = new HashSet<String>();
+    private Set<String> includeTestNames = new HashSet<String>();
+    private Set<String> excludeTestNames = Sets.newHashSet();
```

Comment:
 Should we use consistent style for set creation? Especially if we want to be able integrate this work back to gradle.